lib/game.rb

```diff
@@ -0,0 +1,111 @@
+require 'rspec'
+require 'pry'
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+# from here to war_or_peace runner to play gameplay
+#Order matters
+#how does the match start-end
+class Game
+   attr_reader :deck,
+               :player1,
+               :player2,
+               :turn,
+               :full_deck,
+               :deck1,
+               :deck2
+
+     def initialize
+      @full_deck = new_deck
+      @deck1 = @full_deck[0, 26]
+      @deck2 = @full_deck[26, 26]
+      @player1 = Player.new("Megan", @deck1)
+      @player2 = Player.new("Aurora", @deck2)
```

Comment:
It would be a better practice to create these Player objects outside of the game class, which would allow them to have a dynamic `name` attribute. As such, the Deck objects would probably need to be created outside of this class, too.